src/components/ui/bookmark/bookmark.tsx

```diff
@@ -0,0 +1,28 @@
+type BookmarkProps = {
+  isFavorite: boolean;
+}
+
+const classes = {
+  favorite: {
+    buttonClass: 'place-card__bookmark-button button place-card__bookmark-button--active',
+    spanSign: 'In',
+  },
+  normal: {
+    buttonClass: 'place-card__bookmark-button button',
+    spanSign: 'To',
+  }
+};
+
+function Bookmark({isFavorite}: BookmarkProps): JSX.Element {
+  const bookmarkClasses = classes[isFavorite ? 'favorite' : 'normal'];
+  return (
```

Comment:
return всегда отбиваем пустой строкой
```suggestion
  const bookmarkClasses = classes[isFavorite ? 'favorite' : 'normal'];
  
  return (
```